conda/libmesh/build.sh

```diff
@@ -67,7 +67,8 @@ EOF`
                      --with-vtk-lib=${BUILD_PREFIX}/libmesh-vtk/lib \
                      --with-vtk-include=${BUILD_PREFIX}/libmesh-vtk/include/vtk-${SHORT_VTK_NAME} \
                      --with-methods="opt oprof devel dbg" \
-                     --without-gdb-command
+                     --without-gdb-command \
+                     --with-cxx-std-min=2014
```

Comment:
We somehow need to let` build.sh` call ` libmesh_update_and_rebuild.sh`

Reply:
It might be out the scope of PR

Reply:
I'll play around with this in another PR shortly